Add LICENSE file

uhliarik opened the pull request:
Distribute license text in a separate file.

uhliarik commented:
Hello,

I'm trying to include your nginx module as a package into Fedora distribution [0]. Our Fedora quidelines [1] suggest that license should be distributed in the separate file.

Would it be possible to create and distribute such file within this project in the same way how I propose it in this PR?

[0] https://bugzilla.redhat.com/show_bug.cgi?id=2341923
[1] https://docs.fedoraproject.org/en-US/packaging-guidelines/LicensingGuidelines/#_license_text

zhuizhuhaomeng commented:
@uhliarik would you please remove the license content from the Readme and add a link in the README point to the new license file.

uhliarik commented:
@zhuizhuhaomeng Thanks for your prompt reply. I added a link to the README and moved the whole license text. Let me know if you would like to change anything else.
